gui/src/main.rs

```diff
@@ -152,21 +163,49 @@ impl App for Gui {
                 }
             });
             ui.horizontal(|ui| {
-                if ui.button("Params").clicked() {}
-                if ui.button("Headers").clicked() {}
-                if ui.button("Body").clicked() {}
+                if ui.button("Params").clicked() {
+                    self.config.request_window_mode = RequestWindowMode::PARAMS;
+                }
+                if ui.button("Headers").clicked() {
+                    self.config.request_window_mode = RequestWindowMode::HEADERS;
+                }
+                if ui.button("Body").clicked() {
+                    self.config.request_window_mode = RequestWindowMode::BODY;
+                }
             });
         });
         match self.config.request_window_mode {
             RequestWindowMode::BODY => {
+                TopBottomPanel::top("request_panel")
```

Comment:
Currently this panel seems to be half width? Would be ideal if we could stretch this across the entire width.